src/OSPSuite.Presentation/Presenters/Charts/ChartPresenter.cs

```diff
@@ -61,6 +64,16 @@ public virtual void InitializeAnalysis(TChart chart)
          InitEditorLayout();
       }
 
+      protected Action PostEditorLayout
```

Comment:
This is the method that is called when the Layout is applied to ensure that some columns are always shown if required